app/routers/calendar_grid.py

```diff
@@ -51,6 +51,19 @@ def display(self) -> str:
         """Returns day date inf the format of 00 MONTH 00"""
         return self.date.strftime("%d %B %y").upper()
 
+    def dayview_format(self) -> str:
+        """Returns day date in the format of yyyy-mm-dd"""
+        return self.date.strftime("%Y-%m-%d")
+
+    def weekview_format(self) -> str:
+        """Returns the first day of week in the format of yyyy-mm-dd"""
+        day = self.date
+        if day.strftime("%A") == "Sunday":
```

Comment:
Please add test for this branch

Reply:
added tests to the missing lines - when i open full report it approves. check on my pc also of course.
![image](https://user-images.githubusercontent.com/71097154/109265659-9257c400-780f-11eb-9ee8-fe24640c5c2d.png)

![image](https://user-images.githubusercontent.com/71097154/109265579-72c09b80-780f-11eb-8b2a-301e0fd22765.png)
this line still appers from some reason...

all changes in cov are from files i pulled from the devop branch and not from my changes.